index.html

```diff
@@ -4,12 +4,219 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <link rel="stylesheet" href="styles.css">
-    <title></title>
-    
+    <link rel="stylesheet" href="./styles/styles.css">
+    <link rel="stylesheet" href="https://pro.fontawesome.com/releases/v5.10.0/css/all.css" integrity="sha384-AYmEC3Yw5cVb3ZcuHtOA93w35dYTsvhLPVnYs9eStHfGJvOvKxVfELGroGkvsg+p" crossorigin="anonymous"/>
+    <title>TNW clone</title>  
   </head>
   
   <body>
-    
+    <nav class="nav flex">
+      <div class="flex">
+        <div class="nav-logo">
+          <a href="#">
+            <img src="https://cdn0.tnwcdn.com/wp-content/themes/cyberdelia/assets/img/tnw.svg" alt="The Next Web">
+          </a>
+        </div>
+        <div class="nav-menu flex">
+          <div class="black-bar flex">
+            <ul class="nav-logo-list float-left flex">
+              <li><a href="#">News</a></li>
+              <li><a href="#">Events</a></li>
+              <li><a href="#">Business</a></li>
+              <li><a href="#">AMAs</a></li>
+              <li><a href="#">Spaces</a></li>
+              <li><a href="#">Terms & Conditions</a></li>
+            </ul>
+          
+            <ul class="nav-about flex">
+              <li><a href="#">About</a></li>
+              <li><a href="#">Advertise</a></li>
+              <li><a href="#">Jobs</a></li>
+              <li><a href="#">Contact</a></li>
+            </ul>
+          </div>
+
+          <div class="white-bar flex">
+            <ul class="white-bar-text flex">
+              <li><a href="#">LATEST</a></li>
+              <li><a href="#">HARD WORK</a></li>
+              <li><a href="#">PLUGGED</a></li>
+              <li><a href="#">README</a></li>
+              <li><a href="#">GROTH QUARTERS</a></li>
+          
+              <li><a class="white-bar-sec-text" href="#">SHIFT</a></li>
+              <li><a class="white-bar-sec-text" href="#">NEURAL</a></li>
+            </ul>
+
+            <ul class="nav-icons flex float-right">
+              <li><a href="#" class="search">
+                <img src="./images/computer-icons-pittsburg-button-search-box-button-ede7b4f4cdaaba07daadc3f70cc319f7.png" alt="">
+              </a></li>
+              <li><a href="#" class="menu-icon float-right">
+                <img src="./images/Hamburger_icon.svg.png" alt="">
+              </a></li>
+            </ul>
+          </div>
+
+          
+        </div>
+      </div>
+      <div class="yellow-bar flex">
+        <ul>
+          <li>Calling young digital talent in the Netherlands – Be recognized in the T500 →	</li>
+          
+        </ul>
+      </div>
+    </nav>
+
+    <main>
+      <div class="grid-one">
```

Comment:
This section should be a `header` html semantic tag

Reply:
done